config/crd.yml

```diff
@@ -134,3 +100,22 @@ spec:
     type: date
     description: Time since creation
     JSONPath: .metadata.creationTimestamp
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: internalpackages.install.package.carvel.dev
```

Comment:
note for future: should this be in install or in data?

Reply:
I was thinking these go in to their own group `internal.packaing.dev`, so that the group can just be removed entirely in the future